src/AbstractFixer.php

```diff
@@ -89,21 +89,13 @@ public function supports(\SplFileInfo $file)
         return true;
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function configure(array $configuration = null)
     {
         if (!$this instanceof ConfigurationDefinitionFixerInterface) {
-            throw new \LogicException('Cannot run method for class not implementing `ConfigurationDefinitionFixerInterface`.');
+            throw new \LogicException(sprintf('Cannot configure using Abstact parent, child not implementing "%s".', ConfigurationDefinitionFixerInterface::class));
```

Comment:
Fixed the `::class` usage while cherry-picking. I left the exception message unchanged.